### ISSUE TITLE
error when trying to play random movie

### ISSUE THREAD

**Issue by digiltd** [m0ngr31/kodi-alexa#56]
I am guessing it might be to do with the recent `Add WhatNewMovies intent.` commit?

Let me know if you need more information other than the error message below 

```
 indices must be integers, not str: TypeError
Traceback (most recent call last):
  File "/var/task/wsgi.py", line 1224, in lambda_handler
    return on_intent(event['request'], event['session'])
  File "/var/task/wsgi.py", line 1192, in on_intent
    return one_intent[1](intent_slots)
  File "/var/task/wsgi.py", line 789, in alexa_play_random_movie
    if not 'movies' in movies['result']:
TypeError: list indices must be integers, not str
```

**Comment by jingai** [comment]
Likely.  Will fix in a moment -- working on adding another feature atm.

**Comment by jingai** [comment]
Should be fixed with 94f9e5b2be5a7fdffcbc001596d811172daa1a3e

Let me know if not.

**Comment by digiltd** [comment]
yep, fixed

cheers
